doc/onboarding-extras.md

```diff
@@ -78,6 +78,33 @@ Please use these when possible / appropriate
   git checkout $(git show -s --pretty='%T' $(git show-ref -d $(git describe --abbrev=0) | tail -n1 | awk '{print $1}')) -- test; make -j4 test
   ```
 
+### LTS/Version labels
+
+We use labels to keep track of which branches a commit should land on:
+
+* `dont-land-on-v?.x`
+  * For changes that do not apply to a certain release line
+  * Also used when the work of backporting a change outweighs the benefits
```

Comment:
Is this exclusive with `backport-requested-to`? I thought it would be on a PR that has `backport-requested-to`, because the original PR cannot land, but @MylesBorins said it should not - I guess because the PR is landing in the abstract, it just needs to be backported to make that happen. Nees to be more clear.